ui/components/NodeInputButton.tsx

```diff
@@ -11,14 +21,22 @@ export const NodeInputButton: FC<NodeInputProps> = ({
   dispatchSubmit,
 }) => {
   const handleClick = (e: MouseEvent | FormEvent) => {
-    if (attributes.onclick) {
-      // handle passkeys that bring their own trigger
+    const onClick = attributes.onclick;
+
+    // webauthn registration has a custom handler in webauthn.js to be called
+    // preventing the call of eval with below code
+    if (onClick?.startsWith("window.__oryWebAuthnRegistration(")) {
       e.stopPropagation();
       e.preventDefault();
-      eval(attributes.onclick);
-    } else {
-      void setValue(attributes.value as string).then(() => dispatchSubmit(e));
+
+      const webauthnWindow = window as unknown as WebauthnWindow;
+      const registrationParams = getWebAuthnRegistrationPayload(onClick);
+      webauthnWindow.__oryWebAuthnRegistration(registrationParams);
```

Comment:
question: I see no guard for this cast. Are we sure this will always succeed in this context?

Reply:
If the webauhtn.js script is loaded, the functions will be available. As this is on click, we can be optimistic the script is finished loading at the time of execution. The network request for the script might have failed, leading to an error here. If we guard it with a check if the function on the window object is available and the check is false, I fail to have a better idea that showing a failure -- which should be the same current behaviour.

Reply:
That's ok. What kind of failure would the user see/get in case the function is not available?